raiden/tests/unit/test_pfs_integration.py

```diff
@@ -865,11 +868,12 @@ def test_post_pfs_feedback(query_paths_args):
 
     with patch.object(requests, "post", return_value=request_mock()) as feedback:
         post_pfs_feedback(
+            routing_mode=RoutingMode.PRIVATE,
+            pfs_config=query_paths_args["pfs_config"],
             token_network_address=token_network_address,
             route=route,
             token=feedback_token,
             succesful=False,
-            pfs_config=None,
```

Comment:
does the case exist that there is a pfs_config AND the RoutingMode is Private? Or the other way around? If so, we might need tests for that too

Reply:
It shouldn't be, but in order to keep the diff small I decided to check both.